### PR TITLE
Release azure-cosmos 4.53.2-hotfix

### PR DESCRIPTION
**Pull request by FabianMeiswinkel**

### CODE REVIEW

File: sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosDiagnosticsTest.java [hunk 1]
```diff
@@ -227,7 +227,7 @@ public void gatewayDiagnostics() throws Exception {
         InternalObjectNode internalObjectNode = getInternalObjectNode();
```
Comment (jeet1995) [inline sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosDiagnosticsTest.java:227]: I think we would need to "merge" in the changes related to this PR (the diagnostic enhancement PR for channel acquisition threshold violation) - https://github.com/Azure/azure-sdk-for-java/pull/38416/files#diff-b1526bced1f05d3901ffc6b338ebcc986b8ae37f1238f57f2d8b1579509fbd02

Comment (jeet1995) [reply sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosDiagnosticsTest.java:227]: Same for `RntbdTransportClientTest`.

Comment (jeet1995) [reply sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosDiagnosticsTest.java:227]: Oh sorry - are we porting this PR - https://github.com/Azure/azure-sdk-for-java/pull/38416 - it shows up in the changelog of the backport PR. If not, please ignore / resolve my comments.

**Review by jeet1995 — APPROVED** [review]
LGTM - thanks! I had a question around the porting of the diagnostic enhancement PR but looks good otherwise.

File: sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosDiagnosticsTest.java (hunk 1 above)
Comment (FabianMeiswinkel) [reply sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosDiagnosticsTest.java:227]: Yes - the PR with the channel acquisition changes was cherry-picked intentionally.

Comment (jeet1995) [reply sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosDiagnosticsTest.java:227]: Got it - in that case, I don't see the changes from (#38416) in the hotfix PR.

Comment (FabianMeiswinkel) [reply sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosDiagnosticsTest.java:228]: Good catch :-) Something off when I merged the conflicts I guess. Added the cherry-pick for the missing PR now.

**Pull request merged**